src/Components/CheckSymptomsFlow.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import InfoAlert from './LocationModal/InfoAlert';
+import ShortQuestionAlert from './LocationModal/ShortQuestionAlert';
+
+interface SymptomsFlowProps {
+  location: any;
+  toggleFilter: Function;
+  setFlowFinished: Function;
+}
+
+function CheckSymptomsFlow({
+  location,
+  toggleFilter,
+  setFlowFinished,
+}: SymptomsFlowProps) {
+  const [showNeedHelp, setShowNeedHelp] = useState(false);
+  const [showMapUpdatedAlert, setShowMapUpdatedAlert] = useState(false);
+  const [showNavigateAwayAlert, setShowNavigateAwayAlert] = useState(true);
+  const [
+    showSelfAssessmentCompletedAlert,
+    setShowSelfAssessmentCompletedAlert,
+  ] = useState(false);
+
+  function mapUpdateAgreed() {
+    setShowMapUpdatedAlert(false);
+    setFlowFinished();
+    setShowNavigateAwayAlert(true);
+  }
+
+  function completedAssessment() {
+    setShowSelfAssessmentCompletedAlert(false);
+    setShowNeedHelp(true);
+  }
+
+  function didNotCompleteAssessment() {
+    setShowSelfAssessmentCompletedAlert(false);
+  }
+
+  function needHelp() {
+    toggleFilter('is_collecting_samples', true);
+    setShowNeedHelp(false);
+    setShowMapUpdatedAlert(true);
+  }
+
+  function doesNotNeedHelp() {
+    setShowNeedHelp(false);
+  }
+
+  function navigateAwayAgreed() {
+    setShowNavigateAwayAlert(false);
+    window.open(
+      location.location_contact_url_covid_screening_tool === '' ||
+        location.location_contact_url_covid_screening_tool === null ||
+        location.location_contact_url_covid_screening_tool.length < 4
+        ? 'https://www.apple.com/covid19/'
+        : location.location_contact_url_covid_screening_tool,
+      '_blank'
+    );
+    setShowSelfAssessmentCompletedAlert(true);
+  }
+
+  return (
```

Comment:
Once we get the functionality down, I'd like us to do a bit of UI harmonization such that the prompts here "look and feel" more like the location detail cards.

Reply:
Agreed